src/main.rs

```diff
@@ -407,6 +409,38 @@ fn main() {
         process::exit(0);
     }
 
+    // Language list according to
+    // https://github.com/tldr-pages/tldr/blob/master/CLIENT-SPECIFICATION.md#language
+    let languages = if let Some(ref lang) = args.flag_language {
+        // Language overwritten by console argument
+        vec![lang.clone()]
+    } else if let Ok(lang) = std::env::var("LANG") {
+        let locales = std::env::var("LANGUAGE").unwrap_or_default();
+        let mut locales: Vec<&str> = locales.split(':').collect();
```

Comment:
I'd rename the first variable for clarity:

```suggestion
        let language = std::env::var("LANGUAGE").unwrap_or_default();
        let mut locales: Vec<&str> = language.split(':').collect();
```

Reply:
I think I will just move this method chaining into one assignment for clarity (I don't think having so many variables named "lang", "languages", "locale" and "language" is good either, but good point, this is also weird :D)